app/tests/api/routes/test_LearnRequest.py

```diff
@@ -0,0 +1,43 @@
+from fastapi.testclient import TestClient
+
+
+def test_LearnRequest1(client: TestClient):
```

Comment:
according to PEP8 https://peps.python.org/pep-0008/#function-and-variable-names function names should be snake_case